### PR TITLE
Add https support to deep links

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -67,6 +67,7 @@
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data android:scheme="http" />
+                <data android:scheme="https" />
```
Comment (SUPERCILEX) [inline android/src/main/AndroidManifest.xml]: SO says this [will work](https://stackoverflow.com/a/45782550/4548500), but could you test this with your build already setup just to make sure?

**Comment by SUPERCILEX** [comment]
SGTM

**Comment by SUPERCILEX** [comment]
@phil-lopreiato You think this PR is good?

**Comment by SUPERCILEX** [comment]
@phil-lopreiato Bump

**Comment by SUPERCILEX** [comment]
@phil-lopreiato Bump

**Comment by phil-lopreiato** [comment]
So sorry

**Pull request merged**